lib/rubocop/cop/rspec/capybara/current_path_expectation.rb

```diff
@@ -52,6 +52,19 @@ class CurrentPathExpectation < Base
               $(send nil? :match (str $_)))
           PATTERN
 
+          # Monkey patching for `Style/TrailingCommaInArguments` of RuboCop core
+          # rubocop:disable Style/ClassAndModuleChildren
+          class Style::TrailingCommaInArguments
+            def self.autocorrect_incompatible_with
+              [RSpec::Capybara::CurrentPathExpectation]
+            end
+          end
+          # rubocop:enable Style/ClassAndModuleChildren
```

Comment:
The wording tell me this code was inspired by rubocop-rails which has the "inverse" autocorrect_incompatible_with definition inside the cop, as here.

In rubocop-rspec we have these inverse definitions [here](https://github.com/rubocop/rubocop-rspec/blob/b56d28749734b2985799d2abbf810c91b874ff03/lib/rubocop-rspec.rb#L40-L50). Not sure if that is better or worse, but let’s at least have it be consistent 😄

Reply:
Oops, sorry. I had overlooked that💦
I moved the inverse definitions. Thank you so much!